frontend/ui/widget/inputtext.lua

```diff
@@ -595,11 +595,9 @@ function InputText:onKeyPress(key)
     end
     local handled = true
 
-    if not key["Ctrl"] and not key["Shift"] and not key["Alt"] then
+    if not key["Ctrl"] and not key["Shift"] and not key["Alt"] and not key["ScreenKB"] then
         if key["Backspace"] then
             self:delChar()
-        elseif key["Del"] then
-            self:delNextChar()
```

Comment:
I think the handling of Del should be kept here, under the same conditions as the other ones (no Ctrl, no Shift, no Alt...).
I see no reason to make it generically a special case without these conditions below.

Reply:
mmm, but it would still affect the kindle then. I could add all the conditions to the new `else` if you prefer

Reply:
Can't you just overwrite it later if hasSymKey?

Reply:
Ah never mind, this is in a handling function, not assigning something. But why not simply handle it here?

Reply:
Just add your kindle conditions in there:
```lua
    if not key["Ctrl"] and not key["Shift"] and not key["Alt"] and not key["ScreenKB"] then
        if key["Backspace"] then
            self:delChar()
        elseif key["Del"] then
            if iskindlecrap then -- and write why we do that
                self:delChar()
            else
                self:delNextChar()
            end
        elseif...
```